src/05-objects.js

```diff
@@ -103,7 +127,11 @@ function addValueToObject(object, val) {}
  *  addKeyToObject({ right: false }, "left");
  *  //> { right: false, left: true }
  */
-function addKeyToObject(object, key) {}
+function addKeyToObject(object, key) {
+  object[key] = "val";//does dot notation dont work here????
```

Comment:
It does not for this! Remember dot notation would create that key with the string `key` equal to our value, not the actual value of our key variable.